repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -38,13 +38,15 @@ import org.apache.spark.util.{ParentClassLoader, Utils}
  * Allows the user to specify if user class path should be first.
  * This class loader delegates getting/finding resources to parent loader,
  * which makes sense until REPL never provide resource dynamically.
+ * This class does not set parent classloader since this class loader
```

Comment:
How about rephrase to:
```
[[ClassLoader]] will preferentially load class from parent. Only when parent is null or the load failed, that it will call the overridden `findClass` function. To avoid the potential issue caused by loading class using inappropriate class loader, we should set the parent of ClassLoader to null, so that we can fully control which class loader is used.
```